R/copy-model-from.R

```diff
@@ -191,17 +199,13 @@ copy_control_stream <- function(.parent_model_path, .new_model_path, .overwrite,
       stop("If `.update_model_file` is TRUE, user must specify a `.description` for the new model.")
     }
 
-    # get model id's
-    parent_mod_id <- .parent_model_path %>% get_model_id()
-    new_mod_id <- .new_model_path %>% get_model_id()
-
     # read parent control stream
     mod_str <- .parent_model_path %>% read_file()
 
     # replace the $PROBLEM line(s)
     mod_str <- str_replace(mod_str,
                            "\\$PROB(.|\n)*?\\$",
-                           as.character(glue("$PROBLEM {new_mod_id} {.description}\n\n$")))
+                           as.character(glue("$PROBLEM {.description}\n\n$")))
```

Comment:
Just noting here that the update only happens if user opts in.

Reply:
right, although the default is to update when using `copy_model_from()`. As an interesting note, this makes #134 maybe unnecessary. The idea there was to _not_ replace `$PROB` with the `.description` but instead with the run number. But now the `.description` defaults to the run number, so that gets us to the same place (at least by default). We can talk about that on the 134 issue though.